docs/deploying-airbyte/local-deployment.md

```diff
@@ -28,5 +28,13 @@ Instead of cloning the repo, you can alternatively download the latest Airbyte r
 
 ## Troubleshooting
 
+**I have a Mac with the M1 chip. Is it possible to run Airbyte?**
+
+Some users using Macs with an M1 chip are facing some problems running Airbyte.
+The problem is related with the chip and Docker. [Issue #2017](https://github.com/airbytehq/airbyte/issues/2017) was created to follow up the problem, you can subscribe to it and get updates about the resolution.
+If you can run Airbyte using Apple MacBook with the M1 chip, let us know, the community will appreciate it.
```

Comment:
```suggestion
If you can successfully run Airbyte using a MacBook with the M1 chip, let us know so that we can share the process with the community!
```